src/transformers/modeling_encoder_decoder.py

```diff
@@ -174,6 +174,8 @@ def __init__(
         self.tie_weights()
 
     def tie_weights(self):
+        if self.config.tie_encoder_decoder_word_embeds:
+            self._tie_encoder_decoder_word_embeddings(self.encoder, self.decoder)
```

Comment:
The function `_tie_encoder_decoder_word_embeddings` is actually only relevant for `modeling_encoder_decoder` => Can we move the function to `modeling_encoder_decoder.py` instead?